gempy/utils/logutils.py

```diff
@@ -203,6 +208,14 @@ def config(mode='standard', file_name=None, file_lvl=15, stomp=False):
         log_handler.setLevel(file_lvl)
         rootlog.addHandler(log_handler)
 
+    # Attach additional handlers if provided.
+    if additional_handlers is not None:
+        if isinstance(additional_handlers, list):
```

Comment:
This will fail on some iterables. I think [`collections.abc.Iterable`](https://docs.python.org/3/library/collections.abc.html#collections.abc.Iterable) may be better to check again if keeping `isinstance`. It has some limitations noted in the linked docs. It recommends a more generic:
```python
try:
    iter(additional_handlers)  # Ensures additional_handlers is iterable
    is_iter = True
except TypeError:
    is_iter = False
  ```
  Or something similar, just being explicit with `is_iter` for clarity. Since it's doubtful any weirdly complex iterators would be passed I think it's fine to use `isinstance(..., collections.abs.Iterable)`, but I'll leave that up to you.

Reply:
Alternatively:
```python
try:
    iter(additional_handlers)
except TypeError:
    additional_handlers = [additional_handlers]
finally:
    # ... do things with ensured-iterable additional_handlers ...
```

Reply:
Ooo good catch. I'll make the change.